src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -532,13 +532,16 @@ def cancelWork(self, elementIDs=None, SubscriptionId=None, WorkflowName=None, el
             badWfsCancel = []
             if self.params['PopulateFilesets']:
                 self.logger.info("Canceling work for workflow(s): %s" % (requestNames))
+                #TODO need to find a way to get workflow is aborted or force-complete
+                deleteLogCollect = False
                 from WMCore.WorkQueue.WMBSHelper import killWorkflow
                 for workflow in requestNames:
                     try:
                         myThread = threading.currentThread()
                         myThread.dbi = self.conn.dbi
                         myThread.logger = self.logger
-                        killWorkflow(workflow, self.params["JobDumpConfig"], self.params["BossAirConfig"])
+                        killWorkflow(workflow, self.params["JobDumpConfig"], self.params["BossAirConfig"], 
+                                     deleteLogCollect=deleteLogCollect)
```

Comment:
DeleLogCollect need to be retrieved from workqueue element not parameter passing